azure_arc_servers_jumpstart/docs/arc_updateManagement.md

```diff
@@ -0,0 +1,190 @@
+
+# Configuring Update Management
+
+Update Management uses the Log Analytics agent to collect Windows and Linux server log files and the data collected is stored in a Log Analytics workspace. 
+
+* You will need to create a Log Analytics workspace. For that you can use this [ARM template](../updateManagement/law-template.json) that will create a new Log Analytics Workspace and define the Azure Sentinel solution and enable it for the workspace. 
+
+* First, create a new resource group for the Log Analytics workspace by running the below command, replacing the values in brackets with your own.
+
+    ```bash
+    az group create --name <Name for your resource group> --location <Location for your resources>
+    ```
+    ![](../img/updateManagement/03.png)
+
+* Next, edit the ARM template [parameters file](../updateManagement/law-template.parameters.json), providing a name for your Log Analytics workspace, a location, and a name for your Azure Automation account as shown in the example below:
+
+    ![](../img/updateManagement/05.png)
+
+* To deploy the ARM template, navigate to the [deployment folder](../updateManagement/) and run the below command:
+
+    ```bash
+    az deployment group create --resource-group <Name of your Azure Resource Group> \
+        --template-file law-template.json \
+        --parameters law-template.parameters.json
+    ```
+
+   ![](../img/updateManagement/04.png)
+
+* When the deployment is complete, you should be able to see your Log Analytics workspace with the Update Management solution from the Azure Portal as shown below.
+
+    ![](../img/updateManagement/06.png)
+
+# Onboarding on the Azure Arc enabled server to Update Management
+
+Once you have deployed the Update Management on your Log Analytics workspace, you will need to connect data sources to it. To do this for an Azure Arc enabled server, we must deploy the Log Analytics extension.
+
+* Navigate to the Log Analytics workspace and copy the Workspace ID and Primary key for use later.
+
+    ![](../img/updateManagement/10.png)
+
+* Navigate "Extensions" blade of your Azure Arc connected server as shown in the screenshot below.
+
+    ![](../img/updateManagement/07.png)
+
+* Select the Log Analytics Agent extension.
+
+    ![](../img/updateManagement/08.png)
+
+* Click Create.
+
+    ![](../img/updateManagement/09.png)
+
+* Enter the values for the Workspace ID and key that you copied earlier. and then click through Review + Create.
+
+    ![](../img/updateManagement/11.png)
+
+* The extension should start to deploy and you should see a screen similar to the example below.
+
+    ![](../img/updateManagement/12.png)
+
+* Click on the Solutions blade of the Log Analytics workspace to check the progress of the Update Management assessment.
+
+    ![](../img/updateManagement/13.png)
+
+* It may take several hours for Update Management to collect enough data to show an assessment for your VM. In the screen below we can see the assessment is being peformed.
```

Comment:
I think that this part needs to be more automated or at least offer an automated deployment as an alternative. See code [here](https://github.com/microsoft/azure_arc/blob/e2c61ff6c84107d3ebc42061dd5a209454364c1d/azure_arc_sqlsrv_jumpstart/azure/arm_template/scripts/Arc_Onboarding.ps1#L266)

Reply:
In the example we are starting with an existing Arc enabled VM so deploying the agent is most straightforward for the user by going through the portal. I will point out references to other options to consider for other scenarios.

Reply:
updated, all one arm template now. opted not to give two approaches and instead its just done through a single approach using a single arm template with nested deployments.